anydex/test/wallet/stellar/test_stellar_db.py

```diff
@@ -0,0 +1,116 @@
+import os
```

Comment:
Due to consistency with wallet class names and _test_xlm_provider.py_, file should probably be named _test_xlm_db.py_